@xen-orchestra/rest-api/src/rest-api/rest-api.type.mts

```diff
@@ -36,6 +36,8 @@ type XapiRecordByXapiXoRecord = {
 export type XoApp = {
   tasks: EventEmitter & {
     create: (params: { name: string; objectId?: string; type?: string }) => Task
+    get: (id: string) => Promise<XoTask>
+    list: (opts?: { filter?: string; limit?: number }) => Promise<XoTask[]>
```

Comment:
`opts` in list is not optional.
I think you should keep this typing and update `@xen-orchestra/mixins/tasks.mjs` to handle if no option is passed to list